RELEASE_NOTES.md

```diff
@@ -79,18 +81,18 @@ accompanying blog post.
   users). The mask manager has been enhanced with a new collapsible
   section containing controls for changing masks properties.
 
-  It is also possible to change a properly shared by a group of mask.
+  It is also possible to change a property shared by a group of mask.
```

Comment:
A group of masks? Groupped masks?

Reply:
Both seems good to me. But note that when we will be near the release in a month, we probably will had a better review and correction by @elstoc

Reply:
I'm not even sure what this is _trying_ to say. But as @Nilvus says I will copy-edit the whole thing once the feature freeze is done, so I wouldn't worry about it too much for now.